provider.swagger.yml

```diff
@@ -146,16 +146,17 @@ definitions:
         description: the full url to access the consultation instance
       name:
         type: string
-        description: name of the instance (as it was provided by the client, even if it gets altered later in the instance lifecycle)
+        description: name of the instance
       adminName:
         type: string
-        description: name of the instance admin user (as it was provided by the client, even if it gets altered later in the instance lifecycle)
+        description: name of the instance admin user
       adminEmail:
         type: string
         format: email
-        description: email of the instance admin user (as it was provided by the client, even if it gets altered later in the instance lifecycle)
+        description: email of the instance admin user
```

Comment:
Je cherche à exprimer le fait que le client ne doit pas présumer que le nom de la discussion, le nom de l'admin et son email sont synchrones à tout moment avec ce qui est présent dans l'instance.

Par exemple, le client peut demander la création d'une instance "Test 123", et la renommer ensuite, par le biais de l'interface d'administration de la discussion "Ceci est un exemple modifié". Dans un tel cas, cette API peut répondre avec "Ceci est un exemple modifié" ou "Test 123", et le client ne doit pas présumer que les métadonnées de l'instance sont en phase avec la réalité (cela peut être le cas, mais peut-être pas).

Du coup, je proposerais bien d'ajouter "`(which may or may not be in sync with the running instance)`"